docs/topics/mta-cli/proc_performing-a-live-discovery.adoc

```diff
@@ -7,7 +7,23 @@
 
 You can use a live discovery if you want to determine what is deployed in a certain Cloud Foundry (CF) cluster. For example, you can determine how many applications are in the cluster. You can also use the live discovery if you do not have access to manifest YAML files.
 
-You can run the live discovery for a remote CF instance by using the `mta-cli discover cloud-foundry --use-live-connection --spaces=_<space_name>_` command.
+You can use the `--org`, `--spaces`, and `app-name` flags in that order to explore application manifests in CF clusters. When you use the flags with the `--list-apps` flag, it lists the application manifests by organization(s), by space(s), and  by application name through live discovery. 
+
+{ProductShortName} discovers the application manifests based on the following scenarios:
+
```

Comment:
Does MTA do anything besides discovering the manifests? Does it display or store the information somewhere. If I understand correctly, the `oc` command is `discover`, so it;s clear why you use the verb, but i wonder if it might be clearer to say "discovers and stores" or something like that. Up to you, of course.  If you think the user will understand the full meaning of "discover," then feel free to ignore this comment.